steely/plugins/linden.py

```diff
@@ -113,16 +113,17 @@ def GetMultiple(self, tickers):
     def GetSingle(self, ticker):
         return self.GetMultiple([ticker])[0]
 
+
 TICKER_FETCHERS = [
     IEXTickerFetcher(),  # Fast, hopefully robust
     YahooTickerFetcher(),  # Fast, super flakey
     AlphaVantageTickerFetcher()  # Slow, little flakey
 ]
 
 
-def user_from_name(name, list):
-    for user in list:
-        if user['first_name'] == name:
+def user_from_name(name, user_list):
+    for user in user_list:
+        if user['first_name'].lower() == name.lower():
             return user
```

Comment:
we should add a todo here to handle multiple people with the same first name, in case we ever make a new friend called Cian